service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/CommandManager.java

```diff
@@ -105,10 +105,11 @@ public void configPush(CommandMessage request) {
                 }
             }
         }
-        sendResponse(request);
+        finalizeAndSendResponse(request);
     }
 
-    private void sendResponse(CommandMessage request) {
+    private void finalizeAndSendResponse(CommandMessage request) {
+        request.determineStatus();
```

Comment:
This method is on the object because I think it has better encapsulation since this entire operation takes place within itself.